### PR TITLE
vppapi: Use VPPApiJSONFiles instead of reinventing the wheel.

### PR DESCRIPTION
**Pull request by vifino**
Hey Pim!
I've recently started packaging VPP for NixOS in [my personal repository](https://github.com/vifino/nix-geht).
Along with it `vpp_papi` and now `vppcfg`. However, NixOS being NixOS, it certainly has some non-standard paths.

I figured out that `vpp_papi` already has a JSON API loader, including a utility to search for the proper location.
So I patched that to return the proper Nix Store path.
But to my surprise, `vppcfg` didn't use that class or functionality at all. So I rectified this by removing the custom simplified JSON API file finder and instead made `vppcfg` use the `vpp_papi` provided `VPPApiJSONFiles`.

It shouldn't break any functionality. `VPPApiJSONFiles` is provided for [at least 3 years now](https://github.com/FDio/vpp/commit/edfe2c0079a756f5fb1108037c39450e3521c8bd).

EDIT: Oh, yeah, I also removed `ipaddress` from the dependencies - it's included in Python 3.

### CODE REVIEW

**Review by pimvanpelt — APPROVED** [review]
TIL VPPApiJSONFiles :-)

**Comment by pimvanpelt** [comment]
We have this annoyingly pedantic style/formatting guide - can you run Black on the PR please ?
If you don't have it, the one line diff it proposed is here:
https://github.com/pimvanpelt/vppcfg/actions/runs/3342106344/jobs/5538446818

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
         'importlib-metadata; python_version == "3.8"',
         "yamale",
         "netaddr",
-        "ipaddress",
```
Comment (pimvanpelt) [inline setup.py:12]: Out of curiosity why did you remove this? It's unrelated to the change you intended (even if it might be correct)

**Review by pimvanpelt — APPROVED** [review]
Nice work - Just a nit from black (our python formatter) and a question about a spurious change unrelated to what you're doing with the vpp_json_dir

File: setup.py (hunk 1 above)
Comment (vifino) [reply setup.py:12]: "ipaddress" is actually contained in Python 3 and breaks dependency resolving on NixOS, as there is no such package for Python 3.

**Comment by vifino** [comment]
Hey, I've pushed the change that `black` wanted.

**Comment by pimvanpelt** [comment]
ack on ipaddress - merging now!

**Comment by vifino** [comment]
Nice! I'll probably contribute more as I actually get to use vppcfg. Though I'll probably end up doing a bigger clean up on the `vpp_papi` stuff. Turns out that nobody runs the test cases, they try to test things that have been removed! :)
